From ltsi-dev-bounces@lists.linuxfoundation.org Mon May 28 23:23:29 2012
From: "Shimoda, Yoshihiro" <yoshihiro.shimoda.uh@renesas.com>
Date: Tue, 29 May 2012 15:22:51 +0900
Subject: [LTSI-dev] [PATCH 05/12] usb gadget: clean up FSF boilerplate text
To: ltsi-dev@lists.linuxfoundation.org
Message-ID: <4FC46B3B.4070502@renesas.com>


>From 4c3d87deff2bb6be573b7f9c1c6736e66fad712a Mon Sep 17 00:00:00 2001
From: Klaus Schwarzkopf <schwarzkopf@sensortherm.de>
Date: Fri, 9 Sep 2011 16:10:44 +0200
Subject: usb gadget: clean up FSF boilerplate text

remove the following two paragraphs as they are not needed:

This program is distributed in the hope that it will be useful, but
WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
License for more details.

You should have received a copy of the GNU General Public License along with
this program; if not, write to the Free Software Foundation, Inc.,59
Temple Place - Suite 330, Boston, MA  02111-1307, USA.

Signed-off-by: Klaus Schwarzkopf <schwarzkopf@sensortherm.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
(cherry picked from commit 28c9fc68ebd32d473a8787d05c74e3f39c6c866b)

Conflicts:

    drivers/usb/gadget/amd5536udc.c
    drivers/usb/gadget/amd5536udc.h
    drivers/usb/gadget/at91_udc.c
    drivers/usb/gadget/at91_udc.h
    drivers/usb/gadget/cdc2.c
    drivers/usb/gadget/ci13xxx_msm.c
    drivers/usb/gadget/composite.c
    drivers/usb/gadget/config.c
    drivers/usb/gadget/dbgp.c
    drivers/usb/gadget/dummy_hcd.c
    drivers/usb/gadget/epautoconf.c
    drivers/usb/gadget/ether.c
    drivers/usb/gadget/f_ecm.c
    drivers/usb/gadget/f_eem.c
    drivers/usb/gadget/f_fs.c
    drivers/usb/gadget/f_hid.c
    drivers/usb/gadget/f_loopback.c
    drivers/usb/gadget/f_ncm.c
    drivers/usb/gadget/f_obex.c
    drivers/usb/gadget/f_phonet.c
    drivers/usb/gadget/f_rndis.c
    drivers/usb/gadget/f_sourcesink.c
    drivers/usb/gadget/f_subset.c
    drivers/usb/gadget/f_uvc.c
    drivers/usb/gadget/f_uvc.h
    drivers/usb/gadget/fusb300_udc.c
    drivers/usb/gadget/fusb300_udc.h
    drivers/usb/gadget/g_ffs.c
    drivers/usb/gadget/hid.c
    drivers/usb/gadget/imx_udc.h
    drivers/usb/gadget/inode.c
    drivers/usb/gadget/langwell_udc.c
    drivers/usb/gadget/langwell_udc.h
    drivers/usb/gadget/m66592-udc.c
    drivers/usb/gadget/m66592-udc.h
    drivers/usb/gadget/mass_storage.c
    drivers/usb/gadget/multi.c
    drivers/usb/gadget/ncm.c
    drivers/usb/gadget/ndis.h
    drivers/usb/gadget/net2280.c
    drivers/usb/gadget/net2280.h
    drivers/usb/gadget/omap_udc.c
    drivers/usb/gadget/pch_udc.c
    drivers/usb/gadget/printer.c
    drivers/usb/gadget/pxa25x_udc.c
    drivers/usb/gadget/pxa25x_udc.h
    drivers/usb/gadget/pxa27x_udc.c
    drivers/usb/gadget/pxa27x_udc.h
    drivers/usb/gadget/s3c2410_udc.c
    drivers/usb/gadget/s3c2410_udc.h
    drivers/usb/gadget/storage_common.c
    drivers/usb/gadget/u_ether.c
    drivers/usb/gadget/u_ether.h
    drivers/usb/gadget/uvc.h
    drivers/usb/gadget/uvc_queue.c
    drivers/usb/gadget/uvc_v4l2.c
    drivers/usb/gadget/uvc_video.c
    drivers/usb/gadget/webcam.c
    drivers/usb/gadget/zero.c

Signed-off-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
---
 drivers/usb/gadget/r8a66597-udc.c |   10 ----------
 drivers/usb/gadget/r8a66597-udc.h |   10 ----------
 2 files changed, 0 insertions(+), 20 deletions(-)

diff --git a/drivers/usb/gadget/r8a66597-udc.c b/drivers/usb/gadget/r8a66597-udc.c
index eb1ad72..fe3313a 100644
--- a/drivers/usb/gadget/r8a66597-udc.c
+++ b/drivers/usb/gadget/r8a66597-udc.c
@@ -8,16 +8,6 @@
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation; version 2 of the License.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
- *
  */

 #include <linux/module.h>
diff --git a/drivers/usb/gadget/r8a66597-udc.h b/drivers/usb/gadget/r8a66597-udc.h
index 503f766..a373c4e 100644
--- a/drivers/usb/gadget/r8a66597-udc.h
+++ b/drivers/usb/gadget/r8a66597-udc.h
@@ -8,16 +8,6 @@
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation; version 2 of the License.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
- *
  */

 #ifndef __R8A66597_H__
-- 
1.7.1
_______________________________________________
LTSI-dev mailing list
LTSI-dev@lists.linuxfoundation.org
https://lists.linuxfoundation.org/mailman/listinfo/ltsi-dev

